code/game/objects/items/devices/radio/radio.dm

```diff
@@ -124,8 +124,14 @@
 			<B>[chan_name]</B>: <A href='byond://?src=\ref[src];ch_name=[chan_name];listen=[!list]'>[list ? "Engaged" : "Disengaged"]</A><BR>
 			"}
 
+/obj/item/device/radio/proc/check_traitor_uplink(frequency)
+	if(hidden_uplink)
+		if(hidden_uplink.check_trigger(usr, frequency, traitor_frequency))
+			usr << browse(null, "window=radio")
+			return 1
+
 /obj/item/device/radio/Topic(href, href_list)
-	//..()
+	var/new_frequency
```

Comment:
Sorry for the late response but this still silly
Just define this under the two `if`s that need it